lake-manifest.json

```diff
@@ -1,57 +1,21 @@
 {"version": 4,
- "packagesDir": "./lake-packages",
+ "packagesDir": ".\\lake-packages",
```

Comment:
It was probably an accident to include your local changes to this manifest file.  Can you please revert them?

Reply:
The linter was yelling at me for having an outdated lake manifest, so I pushed an updated one. Though I recognize leaving it unchanged is probably the best way. The manifest file has been acting weird for me the past couple days. Thanks!

Reply:
CI is still yelling at you, I see.  Try merging master (`git merge origin/master`) and resolving any version changes in favour of what's on master, and see if that gets it to compile.

Reply:
```git merge``` tells me that everything's up to date, and I'm seeing other PRs having the same issue (e.g. https://github.com/leanprover-community/mathlib4/pull/1277). Maybe the master manifest needs updating? :) I think I'll ask on the Zulip.

Reply:
I _think_ you can try merging master again once #1279 lands.